apps/frontend/src/index.tsx

```diff
@@ -24,7 +26,7 @@ createRoot(document.querySelector("#root") as HTMLElement).render(
 							{
 								element: (
 									<ProtectedRoute
-										component="Root"
+										component={<AuthWrapper />}
```

Comment:
Should be used inside ProtectedRoute, it should wrap the `component` inside of it